tsdb/engine/tsm1/file_store.go

```diff
@@ -280,11 +280,6 @@ func (f *FileStore) next(key string, t time.Time) ([]Value, error) {
 	defer f.mu.RUnlock()
 
 	for _, fd := range f.files {
-		// Can this file possibly contain this key and timestamp?
```

Comment:
It's actually faster to just do the real search and see if we have any values to search to see if we have values then search again to get the values.